base_layer/core/src/proof_of_work/diff_adj_manager/diff_adj_storage.rs

```diff
@@ -330,12 +330,12 @@ impl DiffAdjStorage {
         );
         match pow.pow_algo {
             PowAlgorithm::Monero => {
-                let target_difficulty = self.monero_lwma.get_difficulty();
+                let target_difficulty = cmp::max(self.min_pow_difficulty, self.monero_lwma.get_difficulty());
```

Comment:
We can't use the same min difficulty for all PoW algos; they're totally different units (and OOM different). 6M on randomX would be astronomical for a single thread on a laptop CPU (roughly 30,000 Hashes/min)

Reply:
Then we should remove this at the get difficulty as well .